docs/0.4/examples/gameroom/walkthrough/index.md

```diff
@@ -2685,3 +2704,205 @@ At this point, the circuit (Alice and Bob's gameroom) is ready. Next, the Acme
 Gameroom daemon must submit the XO smart contract, which is the last step before
 the gameroom is ready for Alice and Bob to play games. See section I-6.6 for an
 explanation of this process.
+
+### I-6. Behind scene 6: Bob accepts Alice's invitation
+
+Most of the steps in this scene are similar to the Bubba Bakery steps described
+earlier. However, one activity is unique — because the Acme node (Alice's node)
+requested the new circuit, it is responsible for submitting the XO smart
+contract that will allow Alice and Bob to play tic tac toe in the new gameroom.
+
+Most of the steps in this scene are similar to the Bubba Bakery steps described
```

Comment:
This section is a repeat of what is above

Reply:
Updated!